### PR TITLE
Library name change

### PR DESCRIPTION
**Pull request by chris-theroux**
### Summary
Library name change and change of any Honeywell reference to Quantinuum.

### Details and comments
Changed online tests to target current Quantinuum emulators.
Also includes change to target quantinuum api. (Referenced here https://github.com/qiskit-community/qiskit-quantinuum-provider/pull/20)

### CODE REVIEW

**Review by mtreinish — COMMENTED** [review]
I haven't had a chance to dive into the code in detail yet, but the big thing I think we should do as part of this rename is make a standalone package/namespace for the provider. So instead of having the package be `qiskit/provider/quantinuum` we should do `qiskit_quantinuum` or `qiskit_quantinuum_provider`. I've been driving a long process of trying to drop the usage of a shared namespace across multiple python packages in Qiskit (see https://github.com/Qiskit/qiskit/issues/559) and we're almost at the point where qiskit will remove the hook points for using the namespace. So while we're renaming things it feels like a good point to create a separate package here.

**Comment by chris-theroux** [comment]
> 

Added the change. Let me know if it looks good.
Thanks!

File: qiskit_quantinuum/VERSION.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+0.2.2
```
Comment (mtreinish) [inline qiskit_quantinuum/VERSION.txt]:
```suggestion
0.3.0
```

File: qiskit_quantinuum/quantinuumbackend.py [hunk 2]
```diff
@@ -24,36 +24,36 @@
...
-from .honeywelljob import HoneywellJob
+from qiskit_quantinuum.exceptions import QiskitError
+from .quantinuumjob import QuantinuumJob
 
 logger = logging.getLogger(__name__)
 
 
-class HoneywellBackend(BackendV1):
-    """Backend class interfacing with a Honeywell backend."""
+class QuantinuumBackend(BackendV1):
```
Comment (mtreinish) [inline qiskit_quantinuum/quantinuumbackend.py:46]: This is a note for the future, but we should look at moving this to `BackendV2` in a follow up PR.

**Review by mtreinish — COMMENTED** [review]
This LGTM, thanks for doing this. I left 2 inline comments one is just a note for the future and not actionable as part of this PR. But we should probably bump the version number to at least 0.3.0

**Review by mtreinish — APPROVED** [review]

**Pull request merged**